src/Controller/StoreApi/OrderApi/OrderApiController.php

```diff
@@ -176,11 +176,7 @@ public function giftcardStateData(SalesChannelContext $context, Request $request
 
```

Comment:
Do we still have `paymentMethod`.`type` in the request body [here](https://github.com/Adyen/adyen-shopware6/pull/454/files#diff-bc795603fdb41f8ae6329ac186aa6df57e44d3603ee3700a98c77971b3660389R163)?

Reply:
Yes, we do have it.